libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -15,5 +15,6 @@ export class ReadingListComponent {
 
   removeFromReadingList(item: ReadingListItem) {
     this.store.dispatch(removeFromReadingList({ item }));
+    this.store.dispatch(undoRemoveFromReadingList({ item }));
```

Comment:
Same here as above! You should not have the ability to undo an action that you do not know has happened successfully!

Reply:
same here, to dispatch multiple actions in removeBook$ effect.